tests/unit/AmalgamatingBusinesses.spec.ts

```diff
@@ -325,7 +321,35 @@ describe('Amalgamating Businesses - add amalgamating business', () => {
     // verify dialog is displayed
     expect(wrapper.vm.errorDialog).toBe(true)
     expect(wrapper.vm.errorDialogTitle).toBe('Unable to add business')
-    expect(wrapper.vm.errorDialogText).toContain('The business you selected could not be added to this filing.')
+    // expect(wrapper.vm.errorDialogText).toContain('An error occurred.')
+
+    vi.resetAllMocks()
+  })
+
+  it('doesn\'t save an amalgamating business - BC - missing auth info', async () => {
+    // open panel
+    await wrapper.setData({ isAddingAmalgamatingBusiness: true })
+
+    // mock services functions
+    vi.spyOn((AuthServices as any), 'fetchAuthInfo').mockImplementation(() => null)
```

Comment:
If null object (ie, exception was thrown due to some other Axios error) verify that error dialog is displayed (lines 349+).